website/settings/defaults.py

```diff
@@ -281,6 +281,7 @@ def parent_dir(path):
 
 # Most Popular and New and Noteworthy Nodes
 POPULAR_LINKS_NODE = None  # TODO Override in local.py in production.
+POPULAR_LINKS_NODE_REGISTRATIONS = None  # TODO Override in local.py in production.
```

Comment:
So much less redundant! I hated that name and meant to go back and change it. So thanks for the reminder!